### DISCUSSION TITLE
SQL Editor Blank Lines

### DISCUSSION THREAD

**Discussion by higgsja** [dbeaver/dbeaver#13959] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version Linux Mint 20
- DBeaver version 21.2.0
- Additional extensions

#### Connection specification:
- Database name and version MariaDb
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  no

#### Describe the problem you're observing:
Leaving blank lines in the SQL editor causes an execution of the SQL to fail. I like white space on long scripts.

**Comment by uslss** [comment, marked as answer]
@higgsja could you try to disable Blank line is statement delimiter option in preferences and check if it helps?
![image](https://user-images.githubusercontent.com/31996417/133937140-4412465d-3232-4300-9da5-e1be6c7373f3.png)

**Comment by higgsja** [comment]
Well done. I did not consider there would be such a preference to look for!

On 9/19/21 12:34 PM, Dziyana wrote:
>
> @higgsja <https://github.com/higgsja> could you try to disable Blank 
> line is statement delimiter option in preferences and check if it helps?
> image 
> <https://user-images.githubusercontent.com/31996417/133937140-4412465d-3232-4300-9da5-e1be6c7373f3.png>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/dbeaver/dbeaver/issues/13945#issuecomment-922509383>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/ABZKGI7SHONJLGX4E6OFXU3UCYNK7ANCNFSM5EIJJQKQ>.
> Triage notifications on the go with GitHub Mobile for iOS 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
> or Android 
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
>
>
-- 
a

--


Thanks!
West Point 1976 H-3
	Join us! <https://hlh-tx.com> 	*Mission:* To measurably improve 
investor performance

" ... techniques shrouded in mystery clearly have value to the purveyor 
of investment advice. After all, what witch doctor has ever achieved 
fame and fortune by simply advising, 'Take two aspirins'?" -- Warren 
Buffett
